Library/Formula/cpp-netlib.rb

```diff
@@ -1,9 +1,9 @@
 class CppNetlib < Formula
   desc "C++ libraries for high level network programming"
   homepage "http://cpp-netlib.org"
-  url "http://downloads.cpp-netlib.org/0.11.2/cpp-netlib-0.11.2-final.tar.gz"
-  version "0.11.2"
-  sha256 "71953379c5a6fab618cbda9ac6639d87b35cab0600a4450a7392bc08c930f2b1"
+  url "https://github.com/cpp-netlib/cpp-netlib/archive/cpp-netlib-0.12.0-rc1.tar.gz"
+  version "0.12.0-rc1"
```

Comment:
Good call! I'm responsible for the only forward project dependency, but having RCs gated to devel makes a lot of sense given the brew devel documentation/guidance.